client/src/pages/Messages/index.tsx

```diff
@@ -50,6 +52,23 @@ export default function Messages(): JSX.Element {
     });
   }, []);
 
+  useEffect(() => {
+    if (socket === undefined) return;
+
+    socket.on('add-online-user', (recipientUserId) => {
+      dispatchConversations({ type: 'ADD_ONLINE_USER', recipientUserId });
+    });
+
+    socket.on('remove-offline-user', (recipientUserId) => {
+      dispatchConversations({ type: 'REMOVE_ONLINE_USER', recipientUserId });

```

Comment:
Small thing but can you align the name of the event and action to match?

Reply:
Sure, I do that